### PR TITLE
[Excalibur] Make population reduced by text dynamic in cases where it increased

### LINKED ISSUE #376: [Excalibur] Make population impact metric cards text dynamic
**What needs to be done? Why does it need to be done?**

When impacted numbers are increased instead of reduced then change the text to `increased by` and use a red color. cc @sychang 

**Additional context**
Screenshot 2020-05-11 11 49 55

Screenshot 2020-05-11 11 49 50

### PR DESCRIPTION
**Pull request by daschi**
## Description of the change

This adds some logic to the impact cards and the overall population section so that we show either `reduced by` for reductions, `increased by` for increased populations and `changed by` for zero differences in the card text. It also changes the color from teal to dark red for increased text. 

I renamed the all the references to `reduction data` to the more general `population impact data` since it could go either way. 

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Closes #376 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/page-response-impact/ReducingR0ImpactMetrics.tsx [hunk 1]
```diff
@@ -1,41 +1,51 @@
...
-        value={<PercentValue value={"50%"} />}
-        subtitle="12% of staff"
+        title={
+          <HospitalBedsTitle
+            title="Hospital beds used at peak"
+            value={percentageBedsUsed}
+          />
+        }
+        value={`${Math.abs(percentageBedsUsed)}%`}
+        subtitle={`${Math.abs(percentageOfStaff)}% of staff`}
```
Comment (jovergaag) [inline src/page-response-impact/ReducingR0ImpactMetrics.tsx:42]: We don't need this subtitle anymore

Comment (jovergaag) [reply src/page-response-impact/ReducingR0ImpactMetrics.tsx:42]: Actually, I'll remove it in my PR. Continue :)

File: src/page-response-impact/utils/ResponseImpactCardStateUtils.tsx [hunk 2]
```diff
@@ -15,32 +18,56 @@ export type reductionCardDataType = {
...
+export type ImpactTitleProps = {
+  title: string;
+  value: number;
+};
+
+export const ImpactTitleSpan = styled.span`
+  color: ${(props) => props.color || Colors.teal};
+`;
+
+export function getSubtitle(valueSign: number) {
```
Comment (jovergaag) [inline src/page-response-impact/utils/ResponseImpactCardStateUtils.tsx:30]: Beautiful.

File: src/page-response-impact/utils/ResponseImpactCardStateUtils.tsx [hunk 3]
```diff
@@ -15,32 +18,56 @@ export type reductionCardDataType = {
...
+  origData: PopulationImpact,
+  currData: PopulationImpact,
+): PopulationImpact {
   return {
     incarcerated: {
       hospitalized:
-        -1 *
-        (currData.incarcerated.hospitalized -
-          origData.incarcerated.hospitalized),
+        origData.incarcerated.hospitalized - currData.incarcerated.hospitalized,
```
Comment (jovergaag) [inline src/page-response-impact/utils/ResponseImpactCardStateUtils.tsx:57]: Good change. Easier to understand this way.

File: src/page-response-impact/ReducingR0ImpactMetrics.tsx (hunk 1 above)
Comment (jovergaag) [reply src/page-response-impact/ReducingR0ImpactMetrics.tsx:42]: Unless I'm looking at the wrong version of the mocks. Looks to me like there is no longer a subtitle, right?

Comment (daschi) [reply src/page-response-impact/ReducingR0ImpactMetrics.tsx:42]: I think you're right if it's the right mocks! @sychang when you have a moment could you confirm that this is the right mock for the bottom cards? It looks like the arrow next to the percentage was removed and the percentages of staff/incarcerated text are removed too. 

image

**Pull request merged**